### DISCUSSION TITLE
No playback black hole 2CH macOS Sonoma 14.5.1

### DISCUSSION THREAD

**Discussion by elfinforest** [ExistentialAudio/BlackHole#804] [General]
Have been using black hole two channel for several years with no issues at all. Within the last two months, I no longer get any playback. QuickTime so it is easy for me to see that it is recording. Input is set on BH2 channel. Output norm setting for everything  is headphone port. Have tried both headphone port and the other option of bh for output. Nothing else that I am aware of on my end has changed. Wondering if anybody else may have had this problem and can help me out. Thank you.

**Comment by devinroth** [comment]
Why did you post this twice?

Devin

**Comment by elfinforest** [comment]
My mistake I was trying to correct the title because it wasn't quite right and didn't realize I had not deleted the original post

**Comment by elfinforest** [comment]
Yep, I think you're right that it is about Quicktime. As crappy as playback is on Audacity, at least it does playback.
